src/components/tracked-controls.js

```diff
@@ -18,7 +19,10 @@ module.exports.Component = registerComponent('tracked-controls', {
     controller: {default: 0},
     id: {type: 'string', default: ''},
     idPrefix: {type: 'string', default: ''},
-    rotationOffset: {default: 0}
+    rotationOffset: {default: 0},
+    // Arm model parameters, to use when not 6DOF. (pose hasPosition false, no position)
+    headElement: {type: 'selector'},
+    defaultHand: {type: 'string', default: 'right'}
```

Comment:
the current version of PR will use gamepad hand if available first (thus property was renamed defaultHand)

you can't hide rotationOffset as it is passed in from the other higher level components such as hand-controls, and it varies based upon the model those higher level components used (NOT constant)

if you are saying you'd like to make defaultHand a constant (e.g. always default to right if not defined) that is possible, but maybe not best?